### PR TITLE
update instructions Terraform install

### LINKED ISSUE #348: Document that 0.13.5 is required and how to download
The docs should detail how to install terraform and a note that a certain version is required.

1. Download the binary with url
2. Add to `~/.local/bin`
3. Add to your path `export PATH=$HOME/.local/bin:$PATH`if not already in your path

### PR DESCRIPTION
**Pull request by laisbsc**
Add @costrouc suggestions on issue #348.

Fixes #348.

### CODE REVIEW

**Comment by danlester** [comment]
@laisbsc Hello, I added a couple of small changes to your update_docs branch based on my experience setting up a QHub on the loud. Hope this is helpful to do on your same branch, rather than getting in your way... Thanks.

**Comment by laisbsc** [comment]
Hey, @danlester! Welcome to the team and this looks wonderful!

Thank you so much! I really have not noticed those incompatibilities. You're a rock star! 🎸

File: docs/source/02_get_started/03_usage.md [hunk 1]
```diff
@@ -168,10 +168,11 @@ pushed to GitHub. Using the URL under the `--repository` flag on the `qhub init`
 
 To add the project to the initialized GitHub repository run:
 ```shell
-git add .github/ .gitignore README.md environments/ image/ infrastructure/ qhub-config.yaml  terraform-state/
+git add .github/ .gitignore environments/ image/ infrastructure/ qhub-config.yaml terraform-state/
+git commit -m"First commit"
```
Comment (viniciusdc) [inline docs/source/02_get_started/03_usage.md:172]:
```suggestion
git commit -m "First commit"
```

**Review by viniciusdc — APPROVED** [review]
Looks great to me!

**Pull request merged**